doc/design/registries/whitelist-registry.md

```diff
@@ -0,0 +1,151 @@
+# Whitelist Registries
+
+vSphere Integrated Containers 1.2 (VIC) added the ability to whitelist registry access in an installed VCH.  When a registry is added to the VCH at installed time, the VCH goes into 'whitelist' mode.  From this point on, the VCH will only allow access to registries in its list of whitelisted registries.  In this mode, users will not be able to access public registries, such as docker hub.
+
+## Specifying Whitelist Registries at Installation
+
+Whitelist registries can be declared during a VCH installation with *vic-machine* parameter, --whitelist-registry.  Two other vic-machine parameters affect whitelist registries, --registry-ca and --insecure-registry.  When whitelist registries are declared during installation, the latter two parameters acts as modifiers to the whitelist registries.
+
+Registry-ca declares additional certificates to verify access to registry servers secured with TLS.  If a registry is declared as a whitelist registry and not an insecure registry (discussed below), the VCH must have access to the server certificate to verify access.  The Photon-based VM that VIC uses has a base set of well-known certificates from public CAs.  If a whitelist registry uses a certificate that is not in that set of well-known certificates, the certificate must be uploaded to the VCH via vic-machine's --registry-ca parameter.
+
+Insecure-registry declares access to a registry server skip the TLS verification.  This modifies the whitelist label and takes precedence.  For instance, if a registry is declared with --whitelist-registry and with --insecure-registry, the VCH will assume the registry is an insecure whitelisted registry.  If the registry is listed with only --whitelist-registry, then the VCH will attempt to verify access using certificates.
+
+If a registry is declared with --insecure-registry but not with --whitelist-registry, vic-machine will add the insecure registries to the list of whitelist registries *IF* at least one whitelist registry was declared.
```

Comment:
Regarding this line:
_.. the VCH will attempt to verify access using certificates_
Does that mean that the VCH will attempt to verify access validating the registry server-certificate. If that is the case, it should be worded that way.